backstory/src/main/java/com/google/sps/perspective/PerspectiveManagerImpl.java

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.perspective;
+
+import au.com.origma.perspectiveapi.v1alpha1.PerspectiveAPI;
+import au.com.origma.perspectiveapi.v1alpha1.models.AttributeType;
+
+/**
+ * An implementation of PerspectiveManager using PerspectiveAPI for analysis.
+ */
+public class PerspectiveManagerImpl implements PerspectiveManager {
+  /** The perspective API used by this instance. */
+  private final PerspectiveAPI perspectiveAPI;
+
+  /** an array of all the types we want analysis on */
+  static final AttributeType[] REQUESTED_ATTRIBUTES = {
+      AttributeType.ATTACK_ON_AUTHOR,
+      AttributeType.ATTACK_ON_COMMENTER, 
+      AttributeType.FLIRTATION, 
+      AttributeType.IDENTITY_ATTACK,
+      AttributeType.INCOHERENT, 
+      AttributeType.INSULT, 
+      AttributeType.LIKELY_TO_REJECT,
+      AttributeType.OBSCENE, 
+      AttributeType.PROFANITY, 
+      AttributeType.SEVERE_TOXICITY,
+      AttributeType.SEXUALLY_EXPLICIT, 
+      AttributeType.SPAM, 
+      AttributeType.THREAT,
+      AttributeType.TOXICITY, 
+      AttributeType.UNSUBSTANTIAL
+    };
```

Comment:
Nit: looks unrelated to this PR.

Reply:
I just added this because it was important for later, and for the overall implementation of PerspectiveManager interface. I'm hopefully not going to have to make any more changes to the interface.